packages/components/src/components/legacy-inline-message/legacy-inline-message.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { isMobile } from '@deriv/shared';
+import Icon from '../icon/icon';
+import Text from '../text';
+import './legacy-inline-message.scss';
+
+const type_to_icon_mapper = {
+    warning: 'IcWalletWarningMessage',
+    information: 'IcWalletInfoMessage',
+    success: 'IcWalletSuccessMessage',
+    error: 'IcWalletDangerMessage',
+};
+
+type TProps = {
+    type?: 'warning' | 'information' | 'success' | 'error';
+    message: React.ReactNode;
```

Comment:
@heorhi-deriv Can we also pass a classname as a prop to mitigate this https://github.com/binary-com/deriv-app/pull/9019#discussion_r1236467625 ?

Reply:
@aum-deriv the reason i decided to not include the classname was the previous comment from @farzin-deriv. that if we let the developer to change the styles using extra classname, there is no sense in our design system, that every developer starts to modify component as he/she wants. Of course there can be the cases where we need to adjust the styles, but we should avoid to do this as much as possible. In my case i need modify them, so i use the base class to do this (but i hide my class into another one that it will not affect to other places).
@farzin-deriv is my understanding of your POV correct?🤔🧐